fe/fe-core/src/main/java/com/starrocks/mysql/privilege/RoleManager.java

```diff
@@ -129,13 +130,14 @@ public void dropUser(UserIdentity userIdentity) {
         }
     }
 
-    public String getRoleName(UserIdentity userIdentity) {
+    public List<String> getRoleNamesByUser(UserIdentity userIdentity) {
```

Comment:
Better to add w/r lock to all of this Class's method

Reply:
I think using locks in Auth class would be enough since RoleManager class only instantiates as a member of Auth class. Thanks to your reminder, I will fix some public methods in Auth class that call interfaces from RoleManager without any lock.